lib/xen.pm

```diff
@@ -30,7 +30,7 @@ use version_utils 'is_sle';
 #   * extra parameters for virsh create / xl create
 
 our %guests = ();
-if (check_var("REGRESSION", "xen-hypervisor") || check_var("REGRESSION", "xen-client")) {
+if (check_var("REGRESSION", "xen")) {
```

Comment:
Good rename of these variables! Actually the variables, like xen-hypervisor, qemu-hypervisor, are also used in feature tests like internal/external snapshot and virtual network management, for condition checking. Suggest to keep them unchanged after this time's adaption :)

Reply:
We can support all variants. Is this solution OK?
```suggestion
if (get_var("REGRESSION") =~ "xen") {
```

Reply:
I am fine with both. Thank you for being so considerate. You can decide which form to use :)

Reply:
The new form should be more universal. I am already running verification run.